roles/install_toggle/meta/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 dependencies:
-  - { role: install_octoprint }
-  - { role: install_framebuffer }
\ No newline at end of file
+  - { role: install_octoprint, when: platform != 'recore' }
```

Comment:
Why does Toggle not depend on OctoPrint when you're not installing on Recore?

Reply:
It does, but it took forever to run the script with running through the whole octoprint installation first. When running the SYSTEM_klipper_octoprint-DEFAULT.yml the roles are all defined there, so we are not really using the dependencies. I think they should be removed for all platforms really.